lib/BootBot.js

```diff
@@ -58,13 +58,32 @@ class BootBot extends EventEmitter {
   }
 
   sendGenericTemplate(recipientId, elements, options) {
+    var image_aspect_ratio = options && options.image_aspect_ratio && options.image_aspect_ratio.toLowerCase();
```

Comment:
We use camelCase for naming variables/functions across the codebase, even if it doesn't match 1:1 with Facebook's param names. For example, we have a `quickReplies` (not `quick_replies`) option in the say method.

We should be consistent and name this method and option `imageAspectRation` instead.